angular-ngrx-scss/src/app/file-viewer/file-explorer-blob/file-explorer-blob.component.spec.ts

```diff
@@ -15,14 +15,14 @@ describe('FileExplorerBlobComponent', () => {
   let fixture: ComponentFixture<FileExplorerBlobComponent>;
   let store: MockStore;
   const initialState: AppState = {
-    repo: {
+    repository: {
       selectedFile: {
         content: 'this is a readme file',
         name: 'starter.dev-github-showcases',
         type: 'file',
         size: 223,
       },
-    } as RepoState,
+    } as RepositoryState,
```

Comment:
We are. The `RepositoryState` contains information for the current repository, and this test appears to be written specifically to test an individual file. So it does the casting to focus directly on the `selectedFile` that it's testing. 

The `AppState` also appears to be needed, as the tests fail without it. I'm guessing because they're global it needs to be wrapped inside to be read properly? But I'm not certain on that.